vita3k/modules/SceCommonDialog/SceCommonDialog.cpp

```diff
@@ -17,12 +17,15 @@
 
 #include "SceCommonDialog.h"
 
-#include <host/app_util.h>
 #include <dialog/types.h>
+
```

Comment:
remove all changes in this file.
again, that is not our convention

Reply:
eu all change ? this file have my implemntentation drm :/

Reply:
i have just run clang format